code/src/components/StartPage.js

```diff
@@ -0,0 +1,128 @@
+import React, { useState } from "react";
+import styled from "styled-components";
+import { fetchGame, game } from "reducers/game";
+import { useDispatch } from "react-redux";
+import music from "../assets/background-music.mp3";
+import sittingKnight from "../assets/sitting-knight.gif";
+import { KnightImg } from "./GamePage";
```

Comment:
again, clever to import the styling from one page to another, will definitely do that instead of repeating myself!